src/createServer.js

```diff
@@ -1,3 +1,65 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const urlArr = req.url.split('?');
+    const queryObject = new URLSearchParams(urlArr[1]);
+    const textToConvert = urlArr[0].slice(1);
+    const caseName = queryObject.get('toCase');
+    const errors = [];
+
+    if (!textToConvert) {
+      errors.push({
+        message:
+          'Text to convert is required. ' +
+          'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    }
+
+    if (!caseName) {
+      errors.push({
+        message:
+          '"toCase" query param is required. ' +
+          'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    } else if (
+      !['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'].includes(caseName)
```

Comment:
You can move all cases in constant. Better way use SUPPORTED_CASES ( or something like that) for constant name.
Also, move all constants (status code, error messages, cases) to upper scope or better way - to separate file.